### ISSUE TITLE
Support for iOS14-style Cupertino date time picker

### ISSUE THREAD

**Issue by maxfrees** [Flutter-Foundation/nest#4]
Support for iOS14-style Cupertino date time picker

**Comment by matthew-carroll** [comment]
Is there any particular reason that this needs to be implemented at the framework level? Could this exist in a package?

**Comment by maxfrees** [comment]
it could exist in a package, but I think the official implementation would be better

**Comment by matthew-carroll** [comment]
There's not really anything "official" other than the actual Apple implementation. If your goal is for this to get into the cupertino package, then I think the first stop is the Flutter repo. Have you already filed an issue for this there? Do you have any feedback from the Flutter team about this?

**Comment by maxfrees** [comment]
https://github.com/flutter/flutter/issues/68177
https://github.com/flutter/flutter/issues/63693


In fact, Flutter has been ignoring this for 5 years, which is why I'm here for help and support for forks

**Comment by matthew-carroll** [comment]
Ok. Before we start looking what we might do for that, I'll make sure the Flutter team has one more chance to address this. @sethladd is this something that's relevant to your Cupertino refresh effort, or will this continue to linger?

**Comment by maxfrees** [comment]
For many requests and things that need to be fixed, many problems with flutter often remain unresolved for many years, and some issues have not been reviewed even if someone has PR

**Comment by matthew-carroll** [comment]
@maxfrees I'm very aware. That's why we're here. But we also need to make sure the Flutter team is given one final chance on each issue that we consider.

**Comment by maxfrees** [comment]
> @maxfrees I'm very aware. That's why we're here. But we also need to make sure the Flutter team is given one final chance on each issue that we consider.

 yes, expect to be standing here to fix it, whether it's in a package or in a way that's built into flutte, like you said, a lot of people are facing issues like this, like the problem with ios 14, which has been delayed until ios18 ... This continues
